tests/vm_test.go

```diff
@@ -1581,9 +1582,9 @@ status:
 			_, _, err := clientcmd.RunCommand(testsuite.GetTestNamespace(nil), k8sClient, "create", "-f", vmJson)
```

Comment:
It seems that this test covers `kubectl`, and not anything that is kubevirt. Do we need it at all? If we do, can we relegate it to somewhere else? (this file is terribly long as it is).

Reply:
It is correct that these tests cover `kubectl`. However, before dropping them we need to verify that all of the cases in this context are covered by other tests (or drop them if they don't make any sense). I opened an issue for it: https://github.com/kubevirt/kubevirt/issues/12709